docs/orchestration/agents/fargate.md

```diff
@@ -40,12 +40,12 @@ $ prefect agent fargate start
 
 The Fargate Agent can be started either through the Prefect CLI or by importing the `FargateAgent` class from the core library. Starting the agent from the CLI will require that the required AWS configuration arguments are set at the environment level while importing the agent class in a Python process will allow you to specify them at initialization.
 
-::: tip Tokens <Badge text="Cloud"/>
-There are a few ways in which you can specify a `RUNNER` API token:
+::: tip API Keys <Badge text="Cloud"/>
+There are a few ways in which you can specify a service account API key:
 
-- command argument `prefect agent fargate start -t MY_TOKEN`
-- environment variable `export PREFECT__CLOUD__AGENT__AUTH_TOKEN=MY_TOKEN`
-- token will be used from `prefect.config.cloud.auth_token` if not provided from one of the two previous methods
+- command argument `prefect agent fargate start -t SERVICE_ACCOUNT_API_KEY`
+- environment variable `export PREFECT__CLOUD__AGENT__AUTH_TOKEN=SERVICE_ACCOUNT_API_KEY`
+- key will be used from `prefect.config.cloud.auth_token` if not provided from one of the two previous methods
```

Comment:
"key" on its own doesn't mean much to me on its own - could you use `API key` everywhere a key is expected?